assets/queries/terraform/alicloud/ros_stack_without_template/metadata.json

```
@@ -0,0 +1,12 @@
+{
+  "id": "92d65c51-5d82-4507-a2a1-d252e9706855",
+  "queryName": "Ros Stack Without Template",
```

```suggestion
  "queryName": "ROS Stack Without Template",
```

(for standard propuses)